src/Symfony/Bridge/Twig/Node/FormThemeNode.php

```diff
@@ -25,15 +25,8 @@ final class FormThemeNode extends Node
     /**
      * @param bool $only
      */
```

Comment:
This comment can be removed now I guess?

Reply:
indeed, fixed